### ISSUE TITLE
Download latest release here isn't working

### ISSUE THREAD

**Issue by dstrickler** [mijit/indigoplugin-slackwebhook#1]
I went to check out your plugin today and saw that the link to download it isn't working. Is the plugin still available?

**Comment by talarczykco** [comment]
Yes, plugin is still available. I updated the link in the README to point to its current location. Good luck, and let me know how it goes!

**Comment by dstrickler** [comment]
Thanks! I'll play with it soon and let you know how I make out.

**Comment by dstrickler** [comment]
It's working! Thanks! :-)

And I love the simplicity of it. Using a webhook is the way to go.
